slick2d-hiero/pom.xml

```diff
@@ -64,12 +69,15 @@
 			</plugin>
 			<plugin>
 				<artifactId>maven-jar-plugin</artifactId>
+				<version>3.0.0</version>
```

Comment:
Could you update the version in the parent POM instead so that it applies to all projects?